xml/System.Web.Script.Serialization/JavaScriptSerializer.xml

```diff
@@ -16,14 +16,19 @@
   <Interfaces />
   <Docs>
     <summary>
-      <see href="https://www.newtonsoft.com/json">Json.NET</see> should be used for serialization and deserialization. Provides serialization and deserialization functionality for AJAX-enabled applications.</summary>
+      For .NET Framework 4.7.2 and later versions, use the APIs in the <see cref="N:System.Text.Json" /> namespace for serialization and deserialization. For earlier versions of .NET Framework, use <see href="https://www.newtonsoft.com/json">Newtonsoft.Json</see>.
```

Comment:
Should we still keep the one-liner description of the type at the end? Up to you.

```suggestion
      For .NET Framework 4.7.2 and later versions, use the APIs in the <see cref="N:System.Text.Json" /> namespace for serialization and deserialization. For earlier versions of .NET Framework, use <see href="https://www.newtonsoft.com/json">Newtonsoft.Json</see>. This type was intended to provide serialization and deserialization functionality for AJAX-enabled applications.
```